server/controllers/robotJobController.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require('mongoose');
+const ssotToRobotParser = require('../services/SsotToRobotParsing/SsotToRobotParser.js');
+const ssotModels = require('../models/singleSourceOfTruthModel.js');
+const robotJobs = [];
```

Comment:
<img width="519" alt="image" src="https://user-images.githubusercontent.com/36270527/111285594-983c0a80-8641-11eb-8987-f6164591c003.png">

Please have a look at those ESLint Warnings. Then it should be fine from my side.

Reply:
For those to go away #157 needs to be merged first

Reply:
Yes, the first warning will go away with #157 and the second one will stay but we have this warning with every Controller